res/css/views/rooms/_ReplyTile.scss

```diff
@@ -60,16 +59,14 @@ limitations under the License.
     // We do reply size limiting with CSS to avoid duplicating the TextualBody component.
     .mx_EventTile_content {
         $reply-lines: 2;
-        $line-height: $font-22px;
 
         text-overflow: ellipsis;
         display: -webkit-box;
         -webkit-box-orient: vertical;
         -webkit-line-clamp: $reply-lines;
-        line-height: $line-height;
 
         .mx_EventTile_body.mx_EventTile_bigEmoji {
-            line-height: $line-height !important;
+            line-height: $font-22px !important; // Same as var(--EventTile_group_line-line-height)
```

Comment:
I think `var(--EventTile_group_line-line-height)` should be kept for future reference about the reason of this line-height specification (it is easy to search if the variable is available).